github_pipeline_observability/pipeline_observability/document-dashboard/github.workflow.json

```diff
@@ -0,0 +1,3532 @@
+          "displayedFields": [
+            "task.runner.name"
+          ],
+          "variant": "single"
+        },
+        "honeycomb": {
+          "colorMode": "color-palette",
+          "colorPalette": "blue",
+          "dataMappings": {
+            "value": "Count"
+          },
+          "displayedFields": [
+            "task.runner.name"
+          ],
+          "legend": {
+            "hidden": false,
+            "position": "auto"
+          },
+          "shape": "hexagon"
+        },
+        "icon": {
+          "icon": "",
+          "showIcon": false
+        },
+        "label": {
+          "label": "Count",
+          "showLabel": false
+        },
+        "singleValue": {
+          "alignment": "center",
+          "autoscale": true,
+          "colorThresholdTarget": "value",
+          "label": "error",
+          "prefixIcon": "",
+          "recordField": "error",
+          "showLabel": true,
+          "trend": {
+            "isVisible": true,
+            "trendType": "auto"
+          }
+        },
+        "table": {
+          "columnTypeOverrides": [],
+          "columnWidths": {},
+          "enableSparklines": false,
+          "hiddenColumns": [],
+          "lineWrapIds": [],
+          "linewrapEnabled": false,
+          "monospacedFontColumns": [],
+          "monospacedFontEnabled": false,
+          "rowDensity": "condensed"
+        },
+        "thresholds": [],
+        "tooltip": {
+          "showCustomFields": false
+        },
+        "unitsOverrides": [
+          {
+            "added": 1727680127964,
+            "baseUnit": "percent",
+            "decimals": 0,
+            "delimiter": false,
+            "displayUnit": null,
+            "identifier": "failureRate",
+            "suffix": "",
+            "unitCategory": "percentage"
+          }
+        ],
+        "valueBoundaries": {
+          "max": "auto",
+          "min": "auto"
+        }
+      }
+    },
+    "19": {
+      "davis": {
+        "davisVisualization": {
+          "isAvailable": true
+        },
+        "enabled": false
+      },
+      "query": "fetch events \n| filter event.kind == \"SDLC_EVENT\" \n| filter event.category == \"pipeline\"  \n| filter event.type == \"run\" \n| filter event.status == \"started\" \n| filter in(vcs.repository.name, $Repository) \n| filter in(cicd.pipeline.name, $Workflow) \n| filter in(cicd.pipeline.run.trigger, $Trigger)  \n| fields start_time, cicd.pipeline.run.id, cicd.pipeline.run.attempt\n| joinNested jobs=[\n  fetch events \n  | filter event.kind == \"SDLC_EVENT\" \n  | filter event.category == \"task\" \n  | filter event.type ==\"build\" \n  | filter event.status == \"started\" \n  | filter event.provider==\"github.com\" \n  | fields cicd.pipeline.run.id, task.run.attempt, start_time \n  | sort start_time asc ],\n  on: {cicd.pipeline.run.id,left[cicd.pipeline.run.attempt] == right[task.run.attempt]} \n| fieldsAdd queueTime = jobs[0][start_time] - start_time ",
```

Comment:
Is a `joinNested` required here? Seems only the first job is taken into account.